Contents/Code/youtube_dl_helper.py

```diff
@@ -35,7 +55,10 @@ def process_youtube(url):
     ...
     """
     youtube_dl_params = dict(
-        outmpl='%(id)s.%(ext)s',
+        logger=youtube_logger,
+        verbose=True,
+        socket_timeout=10,
+        outtmpl=u'%(id)s.%(ext)s',
```

Comment:
```suggestion
```

Not sure what `outtmpl` is, seems like the output filename... but we don't actually download anything.